src/components/TraineeComments.tsx

```diff
@@ -1,59 +1,62 @@
+/* eslint-disable react/button-has-type */
 import React from 'react';
 import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router';
+import { IoIosArrowBack } from 'react-icons/io';
+import { Icon } from '@iconify/react';
+import CommentCard from '../components/CommentCard';
 
 function TraineeComments() {
   const { t } = useTranslation();
+  const navigate = useNavigate();
   return (
-    <div className="bg-neutral-100 dark:bg-dark-bg dark:text-white flex flex-col justify-start items-center  lg:ml-[-32px] md:ml-[-44px] sm:ml-[-24px] sm:pb-80 lg:mt-[-200px] sm:mt-[-200px] md:mt-[-550px] md:py-8 h-full">
-      <div className="w-7/12 flex flex-col md:py-10 sm:py-12 md:mt-0 lg:ml-0 md:ml-[-120px]  sm:mt-44">
-        <div className=" w-7/12 ">
+    <div className="bg-white dark:bg-dark-bg shadow-lg px-5 py-8 rounded-md w-[90%] mx-auto lg:w-[70%] lg:ml-60 mb-10 mt-10">
+      <div className="w-full flex flex-col md:py-10 md:px-20 sm:py-12 md:mt-0 lg:ml-0  sm:mt-44">
+        <div className=" ">
           <h2 className="font-md">{t('Comments')}</h2>
         </div>
-        <div className=" mt-4">
-          <h2>Kenth Ngabo</h2>
-          <p className="text-[#6B7280] dark:text-dark-text-fill">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
-            ad minim veniam
-          </p>
-        </div>
-
-        <div className=" mt-4 ml-24">
-          <h2> John Doe</h2>
-          <p className="text-[#6B7280] dark:text-dark-text-fill ">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit
-          </p>
-          <div>
-            <input
-              className="bg-neutral-100 border-b border-gray-400   dark:bg-dark-bg outline-none"
-              type="text"
-              placeholder="Reply..."
-            />
+        <div className="w-full">
+          <CommentCard />
```

Comment:
Let us have the comment objects and list of them in a JSON, this will be resolved after the introduction of `JSON Server`, but right now we can do with the `json` file which is more clean

Reply:
Solved